lib/jsdom/living/helpers/style-rules.js

```diff
@@ -92,6 +92,12 @@ exports.propertiesWithResolvedValueImplemented = {
     inherited: false,
     initial: "invert",
     computedValue: "computed-color"
+  },
+  // https://drafts.csswg.org/css-fonts-4/#font-family-prop
+  "font-family": {
+    inherited: true,
+    initial: "",
+    computedValue: "as-specified"
```

Comment:
This doesn't appear correct. Per spec it's

> list, each item a string and/or [<generic-family>](https://drafts.csswg.org/css-fonts-4/#generic-family-value) keywords

The appropriate test to add is something like

```html
<div style="font-family: serif,sans-serif,     'Times New Roman',Arial"></div>
```

and then testing that the computed style is `serif, sans-serif, "Times New Roman", Arial` (with the right quotes and spacing).

Making that work correctly would involve introducing some new infrastructure, probably around here? https://github.com/jsdom/jsdom/blob/907daecb80b33069354d8ce7ee9381c39ffa8dd2/lib/jsdom/living/helpers/style-rules.js#L214-L237